Cython/Includes/libcpp/vector.pxd

```diff
@@ -86,3 +86,5 @@ cdef extern from "<vector>" namespace "std" nogil:
         T* data()
         const T* const_data "data"()
         void shrink_to_fit() except +
+        iterator emplace(const_iterator, ...) except +
+        T& emplace_back(...) except +
```

Comment:
This is the definition for C++17, while C++11 would be
```
void emplace_back(...) except +
```
which is however backwards compatible